package.json

```diff
@@ -84,15 +84,16 @@
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-replace": "^5.0.7",
     "@rollup/plugin-terser": "0.4.4",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/react-hooks": "^3.4.2",
-    "@types/react": "^18.2.52",
-    "@vitest/coverage-v8": "^1.2.2",
+    "@testing-library/jest-dom": "^6.4.8",
+    "@testing-library/react": "^12.1.5",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@types/jest": "^29.5.12",
+    "@types/react": "^16.14.60",
+    "@vitest/coverage-v8": "^2.0.5",
     "all-contributors-cli": "^6.26.1",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-plugin-macros": "^2.8.0",
-    "babel-plugin-tester": "^6.5.0",
+    "babel-plugin-macros": "^3.1.0",
+    "babel-plugin-tester": "^7.0.4",
```

Comment:
Updating babel-plugin-preset produces a lot of snapshot changes because of how it formats them so it's best done in a separate PR.